git/password_plan9.go

```diff
@@ -0,0 +1,19 @@
+package git
+
+import (
+	"bitbucket.org/mischief/libauth"
+	"os/user"
+)
+
+func getPassword(url string) (string, error) {
+	user, err := user.Current()
+	if err != nil {
+		return "", err
+	}
+
+	return libauth.Getuserpasswd(
```

Comment:
Thank you, I was wondering how to do this for ghfs. I'll check out mischief's library. I'm guessing that it isn't super complicated and just read/writes the correct files.